example/21_gemm_layernorm/gemm_bias_relu_add_layernorm_xdl_welford_fp16.cpp

```diff
@@ -145,7 +146,7 @@ void host_gemm_layernorm(Tensor<HDataType>& h_m_n,
     auto ref_layernorm_invoker = ref_layernorm.MakeInvoker();
 
     auto ref_layernorm_argument = ref_layernorm.MakeArgument(
-        e_m_n, gamma_n, beta_n, h_m_n, HElementOp{}, {M, N}, {1}, epsilon);
+        e_m_n, gamma_n, beta_n, h_m_n, h_element_op, {M, N}, {1}, epsilon);
```

Comment:
For line 139, suggest to use explicit type conversion like `AccDataType c = static_cast<AccDataType>(c_m_n(m,n))`

Reply:
Type of c_m_n is AccDataType
Tensor<AccDataType> c_m_n(HostTensorDescriptor{M, N});  